wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -4,6 +4,8 @@
 
 import com.android.volley.toolbox.HttpStack;
```

Comment:
We should introduce our HttpStack interface and use it here in order to decouple volley and wasp. Wasp http stack should contains
- setSslSocketFactory
- HttpStack object